src/main/java/com/faforever/api/data/checks/IsClanMembershipDeletable.java

```diff
@@ -9,16 +9,18 @@
 import java.util.Optional;
 
 public class IsClanMembershipDeletable {
-  public static final String EXPRESSION = "clan membership only editable by leader";
-
+  public static final String EXPRESSION = "IsClanMembershipDeletable";
+  
   public static class Inline extends InlineCheck<ClanMembership> {
 
     @Override
     public boolean ok(ClanMembership membership, RequestScope requestScope, Optional<ChangeSpec> changeSpec) {
       Object opaqueUser = requestScope.getUser().getOpaqueUser();
+      int requesterId = ((FafUserDetails) opaqueUser).getId();
       return opaqueUser instanceof FafUserDetails
-          && membership.getClan().getLeader().getId() == ((FafUserDetails) opaqueUser).getId()
-          && membership.getPlayer().getId() != membership.getClan().getLeader().getId();
+          && membership.getPlayer().getId() != membership.getClan().getLeader().getId()
```

Comment:
Oops, my bad, except that IDs in JSONAPI should always be strings :D But I think we can keep them as ints internally, and Elide will convert them as required (like we have been doing it as it seems).